### PR TITLE
Sync dev with 7.3.1

### PR DESCRIPTION
**Pull request by SaikrishnaBairamoni**
# PR Details
This PR is to update Develop with 7.3.1 release changes
## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/Release_notes.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 V2X-Hub Release Notes
 ---------------------------------
+Version 7.3.1, released July 29th, 2022
+--------------------------------------------------------
+**Summary:**
+V2X Hub release version 7.3.1 is a hotfix release for 7.3.0. The fixes primarily occurred during the Implementation of IHP2 Speed Harmonization algorithm in Carma-cloud application.
+
+Enhancements in this release:
+ - Issue 262: Updated CARMA Streets plugin to receive and decode Mobility Path messages into JSON through Kafka.
```
Comment (dan-du-car) [inline docs/Release_notes.md:9]: remove this issue 262

File: docs/Release_notes.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+Bug fixes in this release:
+ - Issue 392: Fixed Large latencies experienced between V2XHub receiving a Traffic Control Request (TCR) and broadcasting corresponding Traffic Control Messages (TCMs).
+ - Issue 394: Fixed V2X Hub TCMs are broadcasted more than the configured maximum quantity, and are broadcasted after an acknowledgement is received.
```
Comment (dan-du-car) [inline docs/Release_notes.md:12]: Remove issue 394. It is still open.

**Review by paulbourelly999 — APPROVED** [review]
Looks good

**Review by dan-du-car — APPROVED** [review]

File: docs/Release_notes.md (hunk 2 above)
Comment (SaikrishnaBairamoni) [reply docs/Release_notes.md]: updated

File: docs/Release_notes.md (hunk 1 above)
Comment (SaikrishnaBairamoni) [reply docs/Release_notes.md]: updated.

**Pull request merged**